js/currencycom.js

```diff
@@ -1330,6 +1331,31 @@ module.exports = class currencycom extends Exchange {
         return this.safeString (types, type, type);
     }
 
+    async fetchDepositAddress (code, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const request = {
+            'coin': currency['id'],
+        };
+        const response = await this.privateGetV2DepositAddress (this.extend (request, params));
+        //
+        //     { "address":"0x97d64eb014ac779194991e7264f01c74c90327f0" }
+        //
+        return this.parseDepositAddress (response);
```

Comment:
Let's pass the second `currency` argument here, and also let's use `safeCurrency` in `parseDepositAddress`:

```JavaScript
currency = this.safeCurrency (undefined, currency);
return {
    'currency': currency['code'],
    ...
};
```

Reply:
Otherwise it will break down upon parsing the structure in python and php.